tests/foreman/ui/test_puppet_classes.py

```diff
@@ -25,26 +23,23 @@ def test_update_positive(self, description):
         class_name = 'foreman_scap_client'
         param_name = 'ca file'
         with Session(self.browser):
-            # Importing puppet classes from puppet-foreman_scap_client
-            # module for update process
-            if self.puppetclasses.search(class_name) is None:
-                self.puppetclasses.import_scap_client_puppet_classes()
-            self.assertIsNotNone(self.puppetclasses.search(class_name))
-            self.puppetclasses.update_class_parameter_description(
-                class_name,
-                param_name,
-                description
-            )
-            self.assertEqual(
-                description,
-                self.puppetclasses.fetch_class_parameter_description(
-                    class_name,
-                    param_name)
-            )
-
-    @data(*generate_strings_list(len1=8))
+            for description in generate_strings_list(len1=8):
```

Comment:
@jyejare ty for the suggestion.  I just reused the old value here.  Lets address this in https://github.com/SatelliteQE/robottelo/issues/2966 to investigate more on why len1=8 was used earlier and we can increase it as necessary.